### ISSUE TITLE
Determine whether device is ISO or ANSI

### ISSUE THREAD

**Issue by SpoinkyNL** [WootingKb/wooting-rgb-sdk#38]
Hello,

For the Wooting Artemis plugin I'd like to be able to detect what kind of physical layout the device has. 
From what I can tell this is possible because Wootility shows the correct layout depending on what firmware you've flashed, either ISO or ANSI.

It would be nice if this could be added to the RGB SDK so that we know whether the extra ISO-only LEDs are present or not.

**Comment by RobertBeekman** [comment]
Hi, sorry to bump this but I was just wondering if there's any time indication for when we might see this added to the SDK, if at all?
We're working towards a stable release of RGB.NET and it would be great if we could detect Wooting physical layout before then :).

**Comment by simon-wh** [comment]
Hi @RobertBeekman thanks for reminding me about this, figured it was about time I got this sorted for ye ;).

You wanna try out this branch? https://github.com/WootingKb/wooting-rgb-sdk/tree/feature/layout-detection
I've got it implemented as a property of the struct returned by `wooting_rgb_device_info`, additionally I added the function `wooting_rgb_device_layout` which pulls it in directly for simplicity. 

I also implemented the changes in my .NET wrapper which can be a reference implementation for yous https://github.com/simon-wh/Wooting.NET/blob/feature/layout/Wooting.NET/RGBControl.cs

Let me know how you get on, I tested it with all the keyboards I have and seemed to be all good for me. If it seems good to you I'll open a PR, get it merged and make a release :)

**Comment by RobertBeekman** [comment]
Hey Simon,

We got good results in RGB.NET/Artemis, I had several people try it out. I used the device info struct since we already leverage that to get the model etc.

So all looks good to me, thanks!

**Comment by PastaJ36** [comment]
@RobertBeekman do you filter for keyboard size (60% / TKL / Full Size) as well?

**Comment by RobertBeekman** [comment]
@PastaJ36 yes we use `WOOTING_DEVICE_TYPE` retrieved via `wooting_rgb_device_info` but that is limited to TKL and full size.

We use this to determine which LED IDs to make available if no valid layout file is loaded.
It's not a huge issue for us as long as we get the correct model number. That's what we use to load the correct layout file. 

In this case I suppose you're thinking of the 60HE?

**Comment by PastaJ36** [comment]
> @PastaJ36 yes we use `WOOTING_DEVICE_TYPE` retrieved via `wooting_rgb_device_info` but that is limited to TKL and full size.
> 
> We use this to determine which LED IDs to make available if no valid layout file is loaded.
> It's not a huge issue for us as long as we get the correct model number. That's what we use to load the correct layout file.
> 
> In this case I suppose you're thinking of the 60HE?

Yeah but I see @simon-wh already set it up to easily add the device. Should we already add it now that you're working on it anyway? We already know the VID/PID and such

**Comment by RobertBeekman** [comment]
Sure thing! Saves us from having to do a separate update later down the line 👍
